package.json

```diff
@@ -76,7 +76,7 @@
 		"prettier": "npm:wp-prettier@^3.0.3",
 		"react": "^18",
 		"react-dom": "^18",
-		"stylelint-config-standard-scss": "^3.0.0",
+		"stylelint-config-standard-scss": "^15.0.1",
```

Comment:
Suggest removing this dependency and replacing the following two lines with `"@wordpress/stylelint-config/scss-stylistic"`.

https://github.com/woocommerce/google-listings-and-ads/blob/3e62818418339bcd509f97fca740548728ebad59/.stylelintrc.json#L3-L4

In this way, it will be more consistent with WordPress coding standards and will not require adding:

https://github.com/woocommerce/google-listings-and-ads/blob/3e62818418339bcd509f97fca740548728ebad59/.stylelintrc.json#L11

Reply:
Removed in https://github.com/woocommerce/google-listings-and-ads/pull/3068/commits/b5f2172b49139bec77dbe34bb0ecd622bf308cc2